### PR TITLE
My Business location insights microservice

### PR DESCRIPTION
**Pull request by mark-walle**

### CODE REVIEW

File: microservices/google-api/google_mybusiness.py [hunk 1]
```diff
@@ -0,0 +1,435 @@
...
+            start_date = start_date.isoformat()
+
+        start_time = start_date + 'T00:00:00Z'
+
+        # if an end_date is defined in the config file, use that date
+        date_api_upper_limit = (
+            datetime.datetime.today().date() - timedelta(days=2)).isoformat()
+        end_date = account['end_date']
+        if end_date == '':
+            # The most recent data available is from two days ago
```
Comment (BJenning) [inline microservices/google-api/google_mybusiness.py]: Is there a reason that the data is available from 18 months to two days ago?

File: microservices/google-api/google_mybusiness.py [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+        # retrieves the request for this location.
+        try:
+            reportInsights = \
+                service.accounts().locations().\
+                reportInsights(body=bodyvar, name=account_uri).execute()
+        except googleapiclient.errors.HttpError as e:
+            logger.info("Request contains an invalid argument. Skipping.")
+            continue
+
+        # # dump reportInsights to a file for manual review
```
Comment (BJenning) [inline microservices/google-api/google_mybusiness.py]: If this is commented out

Comment (BJenning) [inline microservices/google-api/google_mybusiness.py]: If this is commented out, can it just be removed?

File: microservices/google-api/google_mybusiness.py [hunk 3]
```diff
@@ -0,0 +1,435 @@
...
+        # with open(location_uri + 'json', 'w', encoding='utf-8') as f:
+        #     json.dump(reportInsights, f, indent=2)
```
Comment (BJenning) [inline microservices/google-api/google_mybusiness.py]: If this is commented out, can it just be removed?

Comment (mark-walle) [reply microservices/google-api/google_mybusiness.py]: Cleaned up in the last commit.

**Pull request merged**